.drone.yml

```
@@ -21,6 +21,20 @@ steps:
   - name: docker
     path: /var/run/docker.sock
 
+- name: package-images
+  image: rancher/dapper:v0.5.5
+  commands:
+    - docker pull --quiet rancher/hardened-build-base:v1.15.8b5
```

Same 1.15.x as upstream:

```suggestion
    - docker pull --quiet rancher/hardened-build-base:v1.15.14b5
```

---

https://github.com/rancher/rke2/blob/release-1.20/.drone.yml#L18 youre not talking about this upstream?

---

No, but that'll need to change too as part of the release. I'm talking about [upstream kubernetes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#:~:text=is%20now%20built%20using%20go%201.15.14)